R/validate.R

```diff
@@ -26,7 +26,13 @@ validate_input_values <- function(baseline_scenario, shock_scenario, scenario_ge
     input_args[which(names(input_args) %in% c("carbon_price_model"))] <- NULL
   }
 
-  c("baseline_scenario", "shock_scenario", "scenario_geography") %>%
+  validate_values_in_values <- c("baseline_scenario", "shock_scenario", "scenario_geography","carbon_price_model")
```

Comment:
please use a name like vector_character_args (equivalent to vector_numeric_args below)